### PR TITLE
Write more tests for weather and forecast commands. Write tests for w…

### PR DESCRIPTION
**Pull request by drum-IT**
…eather and location API calls. Write test for error util.

### CODE REVIEW

File: cmds/forecast.js [hunk 1]
```diff
@@ -19,6 +20,6 @@ module.exports = async args => {
     return weather.forecast;
   } catch (err) {
     spinner.stop();
-    console.error(err);
+    return error(err, true);
```
Comment (tabuckner) [inline cmds/forecast.js:23]: nice :)

File: tests/unit/error.test.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const error = require("../../utils/error");
+
+test("Error message should be returned", () => {
+  expect(error("error message", false)).toBe("error message");
```
Comment (tabuckner) [inline tests/unit/error.test.js:4]: 👍

File: tests/unit/location.test.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+const locationAPI = require("../../utils/location");
+
+test("String with city and region when location type is location", async () => {
+  const results = await locationAPI("location");
+  expect(results).toEqual(expect.stringMatching(/\w,\s\w/g));
```
Comment (tabuckner) [inline tests/unit/location.test.js:5]: i double checked that we wont get a false positive w/ the regex, and you're good to go 🥇

File: utils/weather.js [hunk 4]
```diff
@@ -12,5 +12,11 @@ module.exports = async location => {
         (select woeid from geo.places(1) where text="${location}")`
     }
   });
-  return results.data.query.results.channel.item;
+  const {
```
Comment (tabuckner) [inline utils/weather.js:15]: nice, if we're going to use stuff like destructuring we might not have support in earlier versions of node. Do we require a minimum version?

If support is not available for earlier versions _and_ we dont want to require a min version we should consider implementing something like babel in a future ticket.

**Review by tabuckner — APPROVED** [review]
approved with one question :)

File: utils/weather.js (hunk 4 above)
Comment (Rejam) [reply utils/weather.js:15]: I'd appreciate using modern javascript so babel would be good.

Comment (drum-IT) [reply utils/weather.js:15]: I'm okay with either approach. I'll add a card to Trello for someone to take this on.

**Pull request merged**